docs/rules/no-restricted-modules.md

```diff
@@ -1,29 +1,35 @@
 # Disallow Node.js modules (no-restricted-modules)
 
-Disallowing usage of specific Node.js modules can be useful if you want to control the available methods, a developer can
-use, to implement a feature.
+A module in Node.js is a simple or complex functionality organized in single or multiple JavaScript files which can be reused throughout the Node.js
```

Comment:
I think `organized in single or multiple JavaScript files` is a bit confusing here. Modules can `require` other modules, but I think it's confusing to refer to other modules `require`d by that module as part of the imported module.